### PR TITLE
#5806 Fix live tests

### LINKED ISSUE #5806: Live tests fail after https://github.com/FlowCrypt/flowcrypt-browser/pull/5795

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed live tests

close #5806 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: test/source/browser/browser-handle.ts [hunk 1]
```diff
@@ -73,13 +73,19 @@ export class BrowserHandle {
   public newPageTriggeredBy = async (t: AvaContext, triggeringAction: () => Promise<void>): Promise<ControllablePage> => {
     const page = (await this.doAwaitTriggeredPage(triggeringAction))!;
     const url = page.url();
-    const parsedUrl = new URL(url);
+    let pathname = url;
+    try {
+      const parsedUrl = new URL(url);
+      pathname = parsedUrl.pathname;
+    } catch (e) {
+      console.error('Error parsing URL:', e.message);
```
Comment (sosnovsky) [inline test/source/browser/browser-handle.ts:81]: live tests pass now, but this error is thrown by each of them.
do you know which URL cause it?

Screenshot 2024-08-08 at 12 05 38

Comment (ioanmo226) [reply test/source/browser/browser-handle.ts:81]: Invalid url which caused above issue is `:`

Comment (sosnovsky) [reply test/source/browser/browser-handle.ts:81]: Got it, but logs look strange - tests passed but there are parsing errors, let's update handling to something like this:
```ts
let pathname: string;
try {
  pathname = new URL(url).pathname;
} catch {
  pathname = url;
}
```

Comment (ioanmo226) [reply test/source/browser/browser-handle.ts]: Sounds good!

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**